alibi_detect/datasets.py

```diff
@@ -290,7 +290,11 @@ def corruption_types_cifar10c() -> List[str]:
     url = 'https://storage.googleapis.com/seldon-datasets/'
     folder = 'cifar10c'
     filetype = 'npy'
-    corruption_types = google_bucket_list(url, folder, filetype)
+    try:
+        corruption_types = google_bucket_list(url, folder, filetype)
+    except RequestException:
+        logger.exception("Could not connect, URL may be out of service")
+        raise
```

Comment:
This exact error with the message is already caught and raised in `google_bucket_list`, not sure we need to do this again here?

Reply:
Thanks, you're totally correct. I have removed the above. 

Think I got confused as was trying to address this issue https://github.com/SeldonIO/alibi-detect/issues/315#issuecomment-1141822714. The actual correction needs to be to add a `pytest.skip` if the above error is raised when `corruption_types_cifar10c` is used within `test_datasets.py`. I'll do this now.